apps/docs/components/assistant-ui/thread.tsx

```diff
@@ -59,9 +60,8 @@ export const Thread: FC = () => {
             />
 
             <ThreadPrimitive.If empty={false}>
-              <div className="aui-thread-viewport-spacer min-h-8 grow" />
+              <ThreadPrimitive.Spacer className="aui-thread-viewport-spacer" />
```

Comment:
does this break backwards compat for existing code updating? what will happen if you use the old primitives

responsibilities: whose responsibility is it to handle the bottom space under the last assistant message? should it be a new component, or the viewport, or the last assistant message?